test/dynamodb/document_client.coffee

```diff
@@ -52,6 +52,39 @@ describe 'AWS.DynamoDB.DocumentClient', ->
         foo: S: 'bar'
       expect(translateInput(input)).to.eql(params)
 
+    it 'translates empty strings', ->
```

Comment:
For all these tests where you're making sure we remove empty inputs, can you also add tests to verify that we don't translate empty strings/sets/buffers if `convertEmptyValues` isn't set?